README.asciidoc

```diff
@@ -0,0 +1,4 @@
+== Running the code

```

Comment:
Setup info missing. What python is required and how to compile segmentation module would be useful